test/integration/targets/git/tasks/checkout-new-tag.yml

```diff
@@ -48,6 +48,9 @@
     that:
       - not update_new_tag|changed
       - "'newtag' in listoftags.stdout_lines"
+  # This test fails on osx.
```

Comment:
I found the problem at [git.py#L640](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/source_control/git.py#L640)

When `remote_url` is a symlink it won't match `unfrackgitpath(repo)` which has gone through `os.path.realpath`.

So this isn't even an OS X issue -- it just so happens that we're hitting it due to the temp files living in `/var/root` which happens to be a symlink to `/private/var/root`.